1.28/pages/04_🧰_Dataframe_toolbar.py

```diff
@@ -21,7 +21,7 @@ def icon(emoji: str):
 icon("🧰")
 
 st.title("Toolbar for dataframes demo", anchor=False)
-st.write("Check out the new toolbar on top of `st.dataframe` and `st.data_editor` that lets you add and delete rows, download your data as a CSV, and search your data. Learn more about [`st.dataframe`](https://docs.streamlit.io/) and [`st.data_editor.`](https://docs.streamlit.io/) (TODO: update links once live)")
+st.write("Check out the new toolbar on top of `st.dataframe` and `st.data_editor` that lets you add and delete rows, download your data as a CSV, and search your data. Learn more about [`st.dataframe`](https://docs.streamlit.io/library/advanced-features/dataframes) and [`st.data_editor`](https://docs.streamlit.io/library/api-reference/data/st.data_editor).")
```

Comment:
Do you intend to link to the Dataframes guide + st.data_editor API reference, or did you want to link to both API references (st.dataframe and st.data_editor)?

Reply:
That is a good point. I linked to the guide for dataframes because it specifically calls out the toolbar. Do we plan to add anything to the dataframes API reference that mentions the toolbar? If not, I think I'll just change the whole sentence to say "check out our guide on dataframes" or something

Reply:
Since the toolbar is just the UI, that documentation will live primarily in the guide. In general, as from maybe a brief mention of some capability in the API reference, I'll put UI reference in separate guides with screenshots, gifs, etc so it doesn't clutter up the API reference.